packages/organizations/origin-organization-irec-api/src/connection/handlers/create-connection.handler.ts

```diff
@@ -129,8 +130,11 @@ export class CreateConnectionHandler implements ICommandHandler<CreateConnection
 
         for (const account of accounts) {
             if (!irecAccounts.some((a) => a.type === account)) {
+                const numberOfChars = 4;
+                const randomCode = randomBytes(numberOfChars / 2).toString('hex'); // 1 byte is 2 characters
```

Comment:
isn't that the other way around? 1 byte -> 8 bits, hex needs 16bits so 2 bytes is 1 character. But the implementation is ok

Reply:
No, in that case it is 1 byte = 2 chars.

![obraz](https://user-images.githubusercontent.com/7525286/164469575-14dc60be-4960-44d0-93f0-1d50e19d28c6.png)

Not 100% sure why